source/tools/testConsistency/ClientApp/app.js

```diff
@@ -0,0 +1,120 @@
+ // Copyright (c) Microsoft Corporation. All rights reserved. 
+ // Licensed under the MIT License. 
+const express = require("express");
+const path = require('path');
+const bodyParser = require('body-parser');
+const querystring = require('querystring');
+const optionsArray = require("./options");
+const postRequest = require("./postRequest");
+
+const getPostRequestPromise = postRequest.getPostRequestPromise;
+
+const app = express();
+app.set('view engine', 'ejs');
+app.set('views', path.join(__dirname, 'views'));
+app.use(bodyParser.urlencoded({extended: false}));
```

Comment:
>{extended: false [](start = 30, length = 16)

Apparently using extended: false has been deprecated - See [https://expressjs.com/en/resources/middleware/body-parser.html](https://expressjs.com/en/resources/middleware/body-parser.html); I would also recommend explaining wiht quick comments your non-default choices in general for parameters. Also good to link to the corresponding doc links for reference for the code reader in your broader class/file comments.